api/metrics-api-jaxrs-1.1/pom.xml

```diff
@@ -124,6 +124,16 @@
   <build>
     <finalName>hawkular-metrics-api-rest-1.1</finalName>
     <plugins>
+      <plugin>
```

Comment:
Why is this needed? Seems strange as the skip and port properties are not even defined here.